core/ResponseDefinitions.php

```diff
@@ -161,6 +161,21 @@
  *      numberOfImages: int,
  *      completionExpectedAt: ?int,
  *  }
+ *
+ * @psalm-type CoreTeam = array{
+ *      id: string,
+ *      name: string,
+ *      icon: string,
+ * }
+ *
+ * @psalm-type CoreTeamResource = array{
+ *       id: int,
+ *       label: string,
+ *       url: string,
+ *       iconSvg: ?string,
+ *       iconURL: ?string,
+ *       iconEmoji: ?string,
```

Comment:
```suggestion
 *       iconSvg?: string,
 *       iconURL?: string,
 *       iconEmoji?: string,
```
IMO this makes the API simpler, but it is a bit more work to implement, so you might not want to go this route.